LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/LW_PerkPack_Integrated.rus

```diff
@@ -43,3 +43,125 @@ DenseSmokeGrenadeEffectDisplayDesc="Густой дым дополнительн
+AbilityDescName="винтовка вектора"
+
+[VektorRifle_MG X2WeaponTemplate]
+TacticalText="<Bullet/> Оружия жнецов очень точны близи, а так же неплохо себя показывают и на дальних дистанциях.\n<Bullet/> Используйте найденные моды чтобы повысить тактическую эффективность Винтовки Тьма."
+
+[Vektor_CG X2WeaponTemplate]
+FriendlyName="\"Flatline\" Арбалет"
+FriendlyNamePlural="\"Flatline\" Арбалеты"
+BriefSummary="Наш финальный дизайн арбалетов на  уровне инопланетных технологий. Болты этого арбалета сопровождаются полем микроплазыми, значитально уменьшающими сопротивление воздуха и позволяя легко проникать через броню."
+TacticalText="<Bullet/> Оружие Гаусса автоматически проникает (игнорирует) 1 ед. брони.<br/><Bullet/> Оружия жнецов очень точны близи, а так же неплохо себя показывают и на дальних дистанциях.\n<Bullet/> Многие Жнецы предпочитают беспшумный композитный арбалет огнестрельному оружию. Действует так же как Винтовка Жнецов технологий Гаусса.\n<Bullet/> Используйте найденные моды чтобы повысить тактическую эффективность Арбалета."
+AbilityDescName="crossbow"
+
+[VektorRifle_BM X2WeaponTemplate]
+TacticalText="<Bullet/> лазменное оружие поражает цель и дальнейшие атаки по ней будут наносить +1 урон.\n<Bullet/> Наиболее передовая винтовка Жнецов в арсенале ХКОМ.\n<Bullet/> Оружия жнецов очень точны близи, а так же неплохо себя показывают и на дальних дистанциях.\n<Bullet/> Используйте найденные моды чтобы повысить тактическую эффективность Винтовки Теневое Копье."
+
+[SPARKChaingun X2WeaponTemplate]
+FriendlyName="Тяжелый пулемет"
+FriendlyNamePlural="Heavy Chainguns"
+BriefSummary="A massive chaingun with devastating anti-personnel capabilities."
+TacticalText="<Bullet/> This weapon's design is based on R. Shen's blueprints from the original alien invasion.\n<Bullet/> It's functionally identical to the Autocannon. "
+AbilityDescName="Тяжелый пулемет"
+
+[SparkRifle_LS X2WeaponTemplate]
+FriendlyName="Тяжелый пулемет"
+FriendlyNamePlural="Тяжелые пулеметы"
+BriefSummary="Тяжелый пулемет стреляет снарядами выского калибра, которые наносят массированный урон."
+TacticalText="<Bullet/> Лазерные оружия получают бонус к точности.\n<Bullet/> Тяжелый пулемет так же по характерстикам как лазерная автопушка."
+AbilityDescName="Тяжелый пулемет"
+
+[SparkRifle_CG X2WeaponTemplate]
+FriendlyName="Оружие распыляющее на частицы"
+FriendlyNamePlural="Дезинтегратор частиц"
+BriefSummary="ДЧ использует многократный усилитель потока сверхзаряженных частиц, эффективных на любой дистанции." 
+TacticalText="<Bullet/> Оружие Гаусса проникает (игнорирует) 1 ед. брони.<br/><Bullet/> Дезинтегратор частиц имеет такой же технологичный уровень как у автопушки Гаусса."
+AbilityDescName="Particle Projection Cannon"
+
+; End Translated
```

Comment:
Since all of this content is now correctly in *Localization/XComGame.int*, could you please remove it from here?

Reply:
done